### PR TITLE
Introduce data item model

### PR DESCRIPTION
**Pull request by miteshvp**

### CODE REVIEW

**Comment by miteshvp** [comment]
@msrb - PTAL

**Comment by miteshvp** [comment]
@msrb - can you take a look. We need this to perform load test for SINGLE data model

File: scripts/entrypoint.sh [hunk 1]
```diff
@@ -73,6 +73,39 @@ else
     sed -i.bckp 's#storage.dynamodb.stores.graphindex.capacity-read=.*#storage.dynamodb.stores.graphindex.capacity-read=25#' ${PROPS}
 fi
 
+if [ -n "${DATA_MODEL}" ] && ( [ "$DATA_MODEL" = "SINGLE" ] || [ "$DATA_MODEL" = "MULTI" ] ); then
+    if grep -i 'storage.dynamodb.stores.edgestore.data-model=' "$PROPS" 1>/dev/null; then
```
Comment (msrb) [inline scripts/entrypoint.sh]: This is going to match also commented out lines.

**Review by msrb — CHANGES_REQUESTED** [review]

**Comment by msrb** [comment]
Just one small issue, otherwise LGTM.

**Comment by miteshvp** [comment]
Thanks. Done.

**Comment by msrb** [comment]
LGTM, thanks ;)

**Pull request merged**